src/app/(auth)/(navigationsBarLayout)/problems/page.tsx

```
@@ -35,111 +46,116 @@ const ProblemsList = () => {
+        <section>
+          <h1 className="text-3xl font-bold mb-2">문제 리스트</h1>
+          <p className="text-gray-400">코딩테스트 문제를 난이도별로 확인하고 도전해보세요</p>
+        </section>
+        <section className="flex flex-col gap-10">
+          <section className="mb-6 p-6 bg-gray-900/50 rounded-[10px] border border-gray-800">
+            <div className="flex flex-row gap-4 items-center w-full">
+              <FilterSelect>
+                <label className="text-base">카테고리 </label>
+                <Select
+                  className="w-full"
+                  title="카테고리"
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)